tables/es-translation.cti

```diff
@@ -42,3 +42,142 @@ capsletter 46
 begcapsword 46-46
 
 # ----------------------------------------------------------------------------------------------
+
+# Emphasis
+emphclass italic
+emphclass underline
+emphclass bold
+emphclass script
+
+begemph bold 35
+endemph bold 35
+begemph italic 35-35
+endemph italic 35-35
+begemph underline 56-35
+endemph underline 56-35
+begemph script 56-35
+endemph script 56-35
+
+# Multiple character definitions
+
+always        \x002F\x002F  6-26-2
+always        \x005C\x005C  5-35-3
+always        \x007C\x007C  456-123
+
+# Numbers followed by letters a-j should be prefixed with dot5
+# Spanish Braille has no contractions so no worries there.
```

Comment:
it does have them, see pr 653

Reply:
@ultraleetj Is there a reason we should worry when these rules are included in the grade 2 table?